Code/GraphMol/Fingerprints/test1.cpp

```diff
@@ -2321,6 +2321,39 @@ void testMACCS() {
     delete m1;
     delete fp1;
   }
+  {
```

Comment:
Is this test new?

Reply:
yeah, PVS/Studio gave some spurious warnings/errors about parts of the MACCS generation code and I wanted to add a test to make sure that they actually were incorrect